hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/FieldOperations.java

```diff
@@ -272,6 +357,27 @@ public int hashCode(GenericRecord record, String fieldName) {
         public void writeFieldFromRecordToWriter(DefaultCompactWriter writer, GenericRecord record, String fieldName) {
             writer.writeCharArray(fieldName, record.getCharArray(fieldName));
         }
+
+        @Override
+        public void writeJsonFormattedField(StringBuilder stringBuilder, AbstractGenericRecord record, String fieldName) {
+            char[] objects = record.getCharArray(fieldName);
+            if (objects == null) {
+                stringBuilder.append("null");
+                return;
+            }
+            stringBuilder.append("[");
+            int objectsSize = objects.length;
+            int j = 0;
+            for (char object : objects) {
+                j++;
+                JsonEscape.writeEscaped(stringBuilder, object);
+                if (j != objectsSize) {
+                    stringBuilder.append(",");
```

Comment:
```suggestion
                    stringBuilder.append(", ");
```
Minor: To make it consistent with the behavior of the Arrays.toString